package.json

```diff
@@ -18,57 +18,52 @@
   },
   "dependencies": {
     "@rails/webpacker": "^3.2.1",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
     "babel-preset-react": "^6.24.1",
     "backbone": "^1.3.3",
     "braintree-web": "^3.28.1",
     "c3": "^0.4.18",
     "classnames": "^2.2.5",
     "clipboard": "^1.7.1",
     "dotenv": "^4.0.0",
+    "empty": "^0.10.1",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.1",
+    "enzyme": "^3.3.0",
+    "faker": "^4.1.0",
+    "flow-bin": "^0.52.0",
+    "husky": "^0.14.3",
     "ismobilejs": "^0.4.1",
-    "jquery": "^3.3.1",
+    "jest": "^22.3.0",
     "jquery-ui": "^1.12.1",
     "jquery-ujs": "^1.2.2",
+    "jquery": "^3.3.1",
     "js-cookie": "^2.2.0",
     "js-yaml": "^3.10.0",
     "libphonenumber-js": "^1.0.7",
+    "lint-staged": "^6.1.0",
     "lodash": "^4.17.4",
     "mailcheck": "^1.1.1",
     "moment": "^2.20.1",
+    "prettier": "^1.10.2",
     "prop-types": "^15.6.0",
     "query-string": "^5.0.1",
-    "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-intl": "^2.4.0",
     "react-onclickoutside": "^6.7.1",
     "react-redux": "^5.0.6",
     "react-select": "^1.2.1",
-    "redux": "^3.7.2",
+    "react": "^16.2.0",
     "redux-thunk": "^2.2.0",
+    "redux": "^3.7.2",
     "selectize": "^0.12.4",
     "speakingurl": "^14.0.1",
-    "urijs": "^1.19.0"
-  },
-  "devDependencies": {
-    "babel-loader": "^7.1.2",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
-    "empty": "^0.10.1",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "enzyme-to-json": "^3.3.1",
-    "faker": "^4.1.0",
-    "flow-bin": "^0.52.0",
-    "husky": "^0.14.3",
-    "jest": "^22.1.4",
-    "lint-staged": "^6.1.0",
-    "prettier": "^1.10.2",
```

Comment:
Since webpacker hooks into `rake assets:precompile` and invokes  yarn (install) with `NODE_ENV=production`, it deletes all devDependencies from node_modules. We need to set NODE_ENV since we need webpack to compile dependencies for production, so this was the only way. There's no downside since we're not copying node_modules into the docker container anyway.